draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -568,36 +524,32 @@ A variety of performance monitoring metrics, including minimum, maximum, average
  |     Configured Threshold: warning, minor, major, critical      |
  |                                                                |
- |   SCTP-Type-2:                                                 |
+ |   SCTG-Type-2:                                                 |
  |     Collection: current                                        |
  |     Configured Threshold: warning, minor, major, critical      |
  |                                                                |
- |   SCTP-Type-3:                                                 |
+ |   SCTG-Type-3:                                                 |
  |     Collection: current, average, min, max                     |
  |   ...                                                          |
- |   SCTP-Type-n:                                                 |
- |----------------------------------------------------------------|
-    // Note: These are just a few examples. More SCTP profile 
-    //       can be defined
+ |   SCTG-Type-n:                                                 |
+ |------------------------------------------------G---------------|
+    // Note: 
+    // These are just a few examples. More SCTG can be defined
 ~~~~
-{: #figure-plug-threshold-definition title="Coherent pluggable Collection and Threshold Profile Definition"}
+{: #figure-plug-threshold-definition title="Coherent pluggable Collection and Threshold Group Definition"}
 
+As mentioned previously, the TCA might be supported for a subset of coherent pluggable monitored attributes. Since it is possible that the coherent pluggable has different capabilities to raise threshold for different monitored attributes, to provide a general solution for threshold definition on coherent pluggable monitored attributes, this draft introduces the concept of "Supported Collection and Threshold Group (SCTG)" shown in {{figure-plug-threshold-definition}} which defines the configurable threshold values and collection types (i.e., the collection of current value, average value, min/max value are supported). The SCTG types will be used in list of coherent pluggables Sheet.
+
+To define the warning, minor, major, critical threshold values for a coherent pluggable monitored attribute, operator should set upper and lower limits that delineate acceptable performance ranges. This ensures that any deviations can be quickly identified and addressed. A rolling window between min-time and max-time should be employed to dynamically adjust these thresholds based on recent data trends, providing a more accurate reflection of current network conditions. By continuously updating the thresholds, network performance can be maintained within optimal parameters, reducing the risk of undetected issues.
+
+Note that sometimes these thresholds are configurable and sometime they are hard-coded. It is also possible that a vendor can support a sub-set and super-set of monitored attributes (for super-set they need to augment the yang model). 
 
```

Comment:
see earlier comment. All of above is reasonable, but where are the gaps and what has to be done to close them? we should be clearer on the scope of this work.